pandas/core/reshape/merge.py

```diff
@@ -1861,8 +1861,8 @@ def _left_join_on_index(left_ax: Index, right_ax: Index, join_keys, sort: bool =
     return left_ax, None, right_indexer
 
 
-def _right_outer_join(x, y, max_groups):
```

Comment:
can you use a partial function instead of doing this in the join_func (which btw simply make a module_level_variable)

Reply:
I am not quite sure what you mean. We should move the content _right_outer_join to a partial func/module level variable? If you meant that, I am not sure how to swap the inputs and outputs in a module level variable?

Reply:
what i mean is instead of having this as a separate function i'd like to inluclue this in the join_func dict above, a way to do that is simply to use a lambda on the value in join_func (for 'right') that does the correct thing

Reply:
Ah, thanks very much. Uses lambda now.